### PR TITLE
[DH-417] Fix error editing companies without a business type

### PR DESCRIPTION
**Pull request by reupen**
Fixes a 500 error editing non-UK and non-registered UK companies with a null business type.

### CODE REVIEW

File: src/apps/companies/controllers/foreign.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const get = require('lodash/get')
```
Comment (teneightfive) [inline src/apps/companies/controllers/foreign.js]: This can be: 

```js
const { get } = require('lodash')
```

Which uses [destructuring](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) and keeps it consistent with how [we import methods](https://github.com/uktrade/data-hub-frontend/blob/1553d8547d02906b23a3da98b62762113c0c3a24/src/apps/auth/controllers/sign-in.js#L2) elsewhere.

File: src/apps/companies/controllers/ukother.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+const get = require('lodash/get')
```
Comment (teneightfive) [inline src/apps/companies/controllers/ukother.js]: Same as above.

File: src/apps/companies/views/edit-foreign.njk [hunk 3]
```diff
@@ -9,7 +9,9 @@
       <div id="business-type" class="key-value__key form-label-bold">Business type
         <a href="/companies/add-step-1" class="button-link">Change</a>
       </div>
-      <div class="key-value__value" aria-labelledby="business-type">Foreign {{businessTypeName}}</div>
+      <div class="key-value__value" aria-labelledby="business-type">
+        Foreign {{businessTypeName or '(type of organisation not known)'}}
```
Comment (teneightfive) [inline src/apps/companies/views/edit-foreign.njk]: Do we need to say _type of organisation not known_? Can we just use _organisation_ as a fallback?

File: test/unit/apps/companies/controllers/ukother.test.js [hunk 4]
```diff
@@ -57,7 +59,7 @@ describe('Company controller, uk other', function () {
     saveCompanyFormStub = sinon.stub().returns(fakeCompanyForm)
     flashStub = sinon.stub()
 
-    this.breadcrumbStub = function () { return this }
+    breadcrumbStub = function () { return this }
```
Comment (teneightfive) [inline test/unit/apps/companies/controllers/ukother.test.js]: Why was this changed? `breadcrumbStub` should exist on the `this` context for all the tests, no?

File: src/apps/companies/controllers/foreign.js (hunk 1 above)
Comment (reupen) [inline src/apps/companies/controllers/foreign.js]: [Not everywhere](https://github.com/uktrade/data-hub-frontend/blob/3d12520a05e82d5bfe98554d28518ea3def3aa09/src/apps/investment-projects/middleware/forms/team-members.js#L1), but will change it :)

File: src/apps/companies/views/edit-foreign.njk (hunk 3 above)
Comment (reupen) [inline src/apps/companies/views/edit-foreign.njk]: Wording was from Mills.

File: test/unit/apps/companies/controllers/ukother.test.js (hunk 4 above)
Comment (reupen) [inline test/unit/apps/companies/controllers/ukother.test.js]: Why should it be on `this`? I made it consistent with `foreign.test.js` so that it works with arrow functions.

Comment (teneightfive) [inline test/unit/apps/companies/controllers/ukother.test.js]: I suppose it fits with the rest of this file for now. We have a few different methods of referencing variables at the moment but would be good to be consistent at some point.

Comment (teneightfive) [inline test/unit/apps/companies/controllers/ukother.test.js]: My concern was that it was adding noise to this fix. The placement of `breadcrumbStub` isn't concerned with this change. 

If you feel it needs to be addressed possibly split it into a separate commit on this PR?

File: src/apps/companies/views/edit-ukother.njk [hunk 5]
```diff
@@ -9,7 +9,9 @@
       <div id="business-type" class="key-value__key form-label-bold">Business type
         <a href="/companies/add-step-1" class="button-link">Change</a>
       </div>
-      <div class="key-value__value" aria-labelledby="business-type">{{businessTypeName}}</div>
+      <div class="key-value__value" aria-labelledby="business-type">
+        {{businessTypeName or 'Type of organisation not known'}}
```
Comment (feedmypixel) [inline src/apps/companies/views/edit-ukother.njk]: is this meant to be capitalised and without brackets? the one above reads:
```(type of organisation not known)```
Where as this read:
```Type of organisation not known```

Comment (reupen) [inline src/apps/companies/views/edit-ukother.njk]: The other one was prefixed with 'Foreign', while this one had no prefix, hence the difference. (Though that does suggest that this one should be prefixed with UK...)
